example/Readme.md

```diff
@@ -0,0 +1,30 @@
+# Examples
```

Comment:
Please convert `example/README.md` and `example/tracker.py` into documentation that we can store inside the `docs` directory.